lab2.js

```diff
@@ -85,13 +136,32 @@ var hello = {
 // speak, and method (that you'll place on the prototype) called
 // sayHello.
 
-function SentientBeing () {
+function SentientBeing(homePlanet, speech) {
+  this.homePlanet = homePlanet;
+  this.speech = speech;
   // TODO: specify a home planet and a language
   // you'll need to add parameters to this constructor
 }
 
 // sb is a SentientBeing object
-function sayHello (sb) {
+SentientBeing.prototype.sayHello = function(sb) {
+    switch (sb.speech) {
+      case 'klingon':
+        console.log(hello[this.speech]);
+      return hello.klingon;
+
+      case 'romulan':
+        console.log(hello[this.speech]);
+      return hello.romulan;
+
+      case 'federation standard':
+        console.log(hello[this.speech]);
+      return 'hello';
+
+      default:
+      return 'no match found';
+    }
```

Comment:
this is a good use case for bracket notation. If the languages on the sentient being match what's being used for parameters in the hello variable, you can do:
console.log(hello[this.language]);
return hello[sb.language];